pages/docs/concepts/asyncapi-document/_section.md

```diff
@@ -0,0 +1,4 @@
+---
+title: AsyncAPI-document
```

Comment:
```suggestion
title: AsyncAPI document
```

Why did you put a dash there? Notice how weird it looks live, too 😂

<img width="300" alt="Screen Shot 2023-08-31 at 6 22 37 PM" src="https://github.com/asyncapi/website/assets/19964402/0fd425fb-421c-4db9-83df-e4784a5b4eea">